styles.css

```diff
@@ -0,0 +1,1110 @@
+html {
+  margin: 0;
+}
+
+body {
+  margin: 0;
+}
+
+ul,
+li {
+  margin-left: 0;
+  padding-left: 0;
+}
+
+/* main content starts here */
+.main-content {
+  margin: 0;
+  padding: 0;
+  display: flex;
+  width: 375px;
+  flex-direction: column;
+  position: relative;
+  background: #fff;
+}
+/* main content ends here  */
```

Comment:
Please do not leave unused code as a commit in your codebase. Unused code as commits makes your code not clear.